docs/source/faq.rst

```diff
@@ -18,6 +34,61 @@ Python does not allow doubles to look into builtin functions and asked them what
           run_test()
 
 
+Patches
+++++++++
+
+How can I make SomeClass(args, kwargs) return my double?
+--------------------------------------------------------
+
+Use ``patch_class`` and ``allow_constructor``::
+
+    from doubles import patch_class, allow_constructor
+
+    import myapp
+
+    def test_something_that_uses_user():
+        patched_class = patch_class('myapp.user.User')
+        allow_constructor(patch_class).and_return('Bob Barker')
```

Comment:
`patched_class`? I'd recommend including test files inside the doc to have tested documentation.